linkerd/config/src/main/scala/io/buoyant/linkerd/config/types/DtabDeserializer.scala

```diff
@@ -11,3 +11,11 @@ class DtabDeserializer extends ConfigDeserializer[Dtab] {
       Dtab.read(_parseString(parser, ctx))
     }
 }
+
+class DtabSerializer extends ConfigSerializer[Dtab] {
```

Comment:
Mostly, I think it's a little surprising that a file called FooDeserializer.scala would contain both `FooDeserializer` and `FooSerializer`. Like the rest of you, I don't feel that strongly about this either way. :)